test_utilities/bin/flutter_test_runner.bat

```diff
@@ -19,7 +19,7 @@ ECHO Running flutter tests from %1
 PUSHD %1
 
 CALL flutter packages get
-CALL flutter analyze
+CALL flutter analyze --fatal-infos
```

Comment:
Did you mean to say `--no-fatal-infos` here and below to be in compliance with https://github.com/flutter/tests#adding-more-tests:

> The tests must pass even if there are analysis 'info' level issues in the code. Generally, this means that if the test performs static analysis, it does so ignoring info level items (i.e., flutter analyze --no-fatal-infos).